src/v2/Apps/Show/Components/ShowHeader.tsx

```diff
@@ -13,8 +13,11 @@ export const ShowHeader: React.FC<ShowHeaderProps> = ({ show, ...rest }) => {
   const { name, startAt, endAt, formattedStartAt, formattedEndAt } = show
 
   const currentTime = useCurrentTime({ syncWithServer: true })
-  // @ts-expect-error STRICT_NULL_CHECK
-  const { formattedTime } = useEventTiming({ currentTime, startAt, endAt })
+  const { formattedTime } = useEventTiming({
+    currentTime,
+    startAt: startAt!,
+    endAt: endAt!,
```

Comment:
**question:** I assume that we were suppressing `STRICT_NULL_CHECK` errors because we couldn't guarantee that `startAt` and `endAt` would be non-null and non-undefined. Has something changed with these props that allows us to ensure that they are non-null and non-undefined?

Reply:
When we use comments like `@ts-expect-error STRICT_NULL_CHECK` we just bypass `STRICT_NULL_CHECK` errors, we don't guarantee that variables are not null/undefined. In this case the changes also used to bypass the error. Just another way to avoid linter errors

Reply:
The errors are still valid in this case though. This hook is expecting a start/end time and I'm sure there are shows in the database that lack one or both. Somewhat problematic here in that we can't conditionally call hooks. Your best bet would be to refactor the hook to deal with the cases when one or both of these are missing (possibly just returning a null for that field, etc)